modules/web/src/core/editor/views/EditorTabs.jsx

```diff
@@ -217,15 +217,15 @@ class EditorTabs extends View {
                 <TabPane
                     tab={
                         <div data-extra="tab-bar-title" className={`tab-title-wrapper ${editor.customTitleClass}`}>
-                            <i className={`fw fw-${icon} tab-icon`} />
-                            {title}
                             <button
                                 type="button"
                                 className="close close-tab pull-right"
                                 onClick={() => this.onTabClose(editor)}
                             >
                                 ×
                             </button>
+                            <i className={`fw fw-${icon} tab-icon`} />
+                            {title}
```

Comment:
Was this change because the close button came before the title ? If so we need to fix it using float:left I think.

Reply:
This is fine I since we use the pull-right for close button.